packages/quantic/force-app/main/default/lwc/quanticQueryError/quanticQueryError.js

```diff
@@ -11,28 +11,36 @@ import goBack from '@salesforce/label/c.quantic_GoBack';
 
 import { errorMap, genericError } from './errorLabels.js';
 
+/** @typedef {import("coveo").QueryError} QueryError */
+/** @typedef {import("coveo").SearchEngine} SearchEngine */
+
+/**
+ * The `QuanticQueryError` component handles fatal errors when performing a query on the index or Search API.
+ * When the error is known, it displays a link to relevant documentation link for debugging purposes.
```

Comment:
`...documentation link for debugging purposes` --> `...documentation for debugging purposes`